compilers/concrete-compiler/compiler/lib/Dialect/FHE/Analysis/MANP.cpp

```diff
@@ -659,9 +660,9 @@ getSqMANP(mlir::concretelang::FHELinalg::SumOp op,
          "Missing squared Minimal Arithmetic Noise Padding for encrypted "
          "operands");
 
-  llvm::APInt operandMANP = operandMANPs[0]->getValue().getMANP().value();
-
-  return APIntWidthExtendUMul(noiseMultiplier, operandMANP);
+  result = operandMANPs[0]->getValue().getMANP().value();
+  result = APIntWidthExtendUMul(noiseMultiplier, result);
+  return result;
```

Comment:
why isn't it happy with `return APInt...`?

Reply:
I just let that when I try and error. But you are right.